src/app/core/error-handler/error-handler.service.ts

```diff
@@ -1,13 +1,24 @@
 import { HttpErrorResponse } from '@angular/common/http'
 import { Injectable } from '@angular/core'
 import { throwError } from 'rxjs'
+import { CookieService } from 'ngx-cookie-service'
+import { PlatformInfoService } from 'src/app/cdk/platform-info'
+import { take } from 'rxjs/internal/operators/take'
 
 @Injectable({
   providedIn: 'root',
 })
 export class ErrorHandlerService {
-  constructor() {}
+  browserSupport = 'unchecked'
+  constructor(
+    private _platform: PlatformInfoService,
+    private _cookie: CookieService
+  ) {
+    this.checkBrowser()
+    this._cookie.set('XSRF-TOKEN', 'test')
```

Comment:
I think you should put a todo tag or comment here to remove it later. In case of someone else continues with te fix

Reply:
oh god this was just for testing... removing